data/faq.html

```diff
@@ -66,7 +66,8 @@
         font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
       }
 
-      html {
+
+     html {
```

Comment:
Thanks! It would be great to keep the same indentation as before. And sign commits.